lib/api/contacts.js

```diff
@@ -3,3 +3,71 @@ export const queryEmails = (data) => ({
     method: 'get',
     data
 });
+
+export const getContacts = ({ Page = 0, PageSize = 1000, LabelID = '' }) => ({
+    url: `contacts/?${PageID},${PageSize},${LabelID}`,
+    method: 'get',
+});
+
+export const getContactExport = ({ Page = 0, PageSize = 1000, LabelID = '' }) => ({
+    url: `contacts/export${PageID},${PageSize},${LabelID}`,
+    method: 'get',
+    data: { Code, Contacts, Total },
+});
+
+export const getContact = ({ contactID }) => ({
+    url: `contacts/${contactID}`,
+    method: 'get',
+});
+
+export const addContacts = ({ Contacts, Overwrite, Labels }) => ({
+    url: 'contacts',
+    method: 'post',
+    data: { Contacts, Overwrite, Labels },
+});
+
+export const updateContact = (contactID, { Cards }) => ({
+    url: `contacts/?${contactID}`,
```

Comment:
```suggestion
    url: `contacts/${contactID}`,
```

Reply:
Ok. But then why is the '?' in the API specs. I noticed sometimes it's there sometimes it's not. Does it mean something?

Reply:
It's a mistake in the API doc I think if it's present for this route.